src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[]{figureSupplier.getRandomFigure(),
+                figureSupplier.getRandomFigure(),
+                figureSupplier.getDefaultFigure(),
+                figureSupplier.getDefaultFigure()};
+        for (Figure figure : figures) {
+            figure.draw();
+            figure.getArea();
+        }
```

Comment:
better create an array of fixed length (constant), then fill it in the for loop (if the current index is less the half of length- random, else - default). Then call draw(). Why to call getArea() at all?